olive/strategy/search_parameter.py

```diff
@@ -313,9 +313,10 @@ def json_to_search_parameter(json: Dict[str, Any]) -> SearchParameter:
     if search_parameter_type == "Categorical":
         return Categorical(json["support"])
     if search_parameter_type == "Conditional" or search_parameter_type == "ConditionalDefault":
-        stop_condition = lambda x: (  # noqa: E731
-            isinstance(x, dict) and x.get("olive_parameter_type") == "SearchParameter"
-        )
+
+        def stop_condition(x):
+            return isinstance(x, dict) and x.get("olive_parameter_type") == "SearchParameter"  # noqa: E731
```

Comment:
> # noqa: E731 [](http://example.com/codeflow?start=94&length=13)

no need this one?

Reply:
Good catch. This line is auto-formatted from my local linter plugin. But somehow the `noqa` was not removed.